library/src/math/stretch.rs

```diff
@@ -142,8 +143,8 @@ fn assemble(
             advance += ratio * (max_overlap - min_overlap);
         }
 
-        let fragment = GlyphFragment::with_id(ctx, base.c, part.glyph_id, base.span);
-        selected.push((fragment, advance));
+        base.set_id(ctx, part.glyph_id);
+        selected.push((base.clone(), advance));
```

Comment:
I might be wrong here, but just to make sure: Wouldn't it be better to first clone the base and then run `set_id` on the clone, instead of mutating it, as `base` is used after the loop? Or would that break things?

Reply:
Oh, yeah, my mistake, sorry.

Reply:
Actually, that fixed the affected "underbrace", thanks!